src/app/mytools/sidebar-accordion/sidebar-accordion.component.ts

```diff
@@ -3,38 +3,48 @@ import { DockstoreTool } from 'app/shared/swagger';
 import { Observable } from 'rxjs';
 import { ToolQuery } from '../../shared/tool/tool.query';
 import { OrgToolObject } from '../my-tool/my-tool.component';
+import { KeyValue } from '@angular/common';
+import { MetadataService } from '../../shared/swagger/api/metadata.service';
 
-interface groupEntriesBySource {
+interface GroupEntriesByRegistry {
   groupEntryInfo: OrgToolObject<DockstoreTool>[];
-  sourceControlTitle: string;
+  registryTitle: string;
 }
 
 @Component({
   selector: 'app-sidebar-accordion',
   templateUrl: './sidebar-accordion.component.html',
-  styleUrls: ['./sidebar-accordion.component.scss'],
+  styleUrls: ['./sidebar-accordion.component.scss', '../../shared/styles/my-entry-sidebar.scss'],
 })
 export class SidebarAccordionComponent implements OnInit, OnChanges {
   @Input() openOneAtATime;
   @Input() groupEntriesObject: OrgToolObject<DockstoreTool>[];
   @Input() refreshMessage;
   public toolId$: Observable<number>;
+  private dockerRegistryMap = [];
```

Comment:
Similar to Charles' comment for `sourceControlMap`. You can probably get rid of this property since you only use it once on line 31